src/ui/public/vislib/lib/axis/axis_labels.js

```diff
@@ -108,7 +108,7 @@ export function VislibAxisLabelsProvider() {
 
           if ((startPos + halfSize) < myPos && maxSize > (myPos + halfSize)) {
             startPos = myPos + halfSize;
-            return this.innerHTML;
+            return this.innerHTML || (new XMLSerializer()).serializeToString(this.firstChild);
```

Comment:
Could we extract this to a method `xmlSerializeToString` which also uses a class field to store the `XMLSerializer` instead of creating it on every call?

Reply:
we could ... not sure how far to go with this as its IE11 specific spot fix ...